File: 19 -API Rest/xkcd.php
<?php

echo file_get_contents('https://xkcd.com/info.0.json').PHP_EOL;

#Use php xkcd.php | jq to get better results

$json = file_get_contents('https://xkcd.com/info.0.json');
#To get data as an array
$data = json_decode($json, true);

echo $data['img'];

File: 19 -API Rest/xkcd.py
from urllib.request import urlopen
import json

base_url = "https://xkcd.com/info.0.json"

# To convert bytes in text
data = urlopen(base_url).read().decode('UTF-8')

# json.loads to decode in text
data_json_obj = json.loads(data)

print(data_json_obj["img"])